packages/analyzer/src/javascript/class-scanner.ts

```diff
@@ -541,7 +548,8 @@ class PrototypeMemberFinder implements Visitor {
       description,
       sourceRange: this._document.sourceRangeForNode(node)!,
       warnings: [],
-      astNode: node,
+      astNode:
+          {language: 'js' as 'js', containingDocument: this._document, node},
```

Comment:
Why is the cast to `as 'js'` necessary here?

Reply:
It's not clear that this is an `AstNodeWithLanguage`, and TypeScript's type inference doesn't want to infer a too-specific type. For example, should the function `() => false` have the type `() => boolean` or `() => false`? TypeScript will default to `() => boolean` unless there's a signal from the author that the more specific type is preferred. One way to do that is to write `() => false as false`, another is to assign it directly to a value which expects the type `() => false`.

Reply:
Ah I see. Thanks for the explanation!